core/build-image.sh

```diff
@@ -119,18 +119,45 @@ buildah commit "${container}" "${repobase}/${reponame}"
 buildah rm "${container}"
 images+=("${repobase}/${reponame}")
 
+echo "Building a patched rclone binary..."
+bcontainer=$(buildah from docker.io/library/alpine:3.18.4)
+buildah run ${bcontainer} sh <<'EOF'
+apk add --no-cache binutils-gold go patch curl
+set -e
+mkdir -p /srv/build
+cd /srv/build
+rm -rf rclone-*
+curl -L https://github.com/rclone/rclone/archive/refs/tags/v1.62.2.tar.gz | tar xvz
+cd rclone-*
+echo "Apply patch for systemd-activation to $(pwd)..."
+curl -L https://github.com/rclone/rclone/commit/0f15efae616354cc887063835c8dfa510488f9a6.patch | patch -p1
+export GO111MODULE=on
+export CGO_ENABLED=0
+go build
```

Comment:
I think this will be hardly maintainable in the long run.
As an example, when we are going to bump the rclone version, the path could be invalid for the new release.
I also do not see any real advantage on having socket activation: you can just spare few megabytes of RAM when the backup is not running.